Source/settings/mods.ts

```diff
@@ -255,6 +255,34 @@ module FullScreenPokemon {
                         }
                     }
                 }
+            },
+            {
+                name: "Scaling Levels",
+                enabled: false,
+                events: {
+                    onModEnable: function (mod: ModAttachr.IModAttachrMod): void {
+                        return;
+                    },
+                    onModDisable: function (mod: ModAttachr.IModAttachrMod): void {
+                        return;
+                    },
+                    onBattleReady: function (mod: ModAttachr.IModAttachr, eventName: string, battleInfo: IBattleInfo): void {
```

Comment:
Also, can you comment these functions? This wasn't a requirement before but they're getting complicated to the point where it's necessary.